src/WebJobs.Script/Description/DotNet/FSharp/FSharpCompilationService.cs

```diff
@@ -196,13 +172,17 @@ public ICompilation GetFunctionCompilation(FunctionMetadata functionMetadata)
                     var assembly = Assembly.Load(assemblyBytes, pdbBytes);
                     assemblyOption = FSharpOption<Assembly>.Some(assembly);
                 }
+                else
+                {
+                    _traceWriter.Verbose($"F# Compilation failed with arguments: { string.Join(" ", otherFlags) }");
```

Comment:
The actual compilation/diagnostics errors are already logged and reported back. This gives us some additional details in verbose mode to show the entire set of arguments passed to the compiler.